sormas-app/app/src/main/res/layout/row_vaccination_reduced_list_item_layout.xml

```diff
@@ -0,0 +1,205 @@
+            >
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/vaccination_uuid"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="sans-serif-medium"
+                    android:textColor="@color/listActivityRowLeastText"
+                    android:textSize="@dimen/listActivityRowTertiaryTextSize"
+                    app:defaultValue='@{@string/value_uuid_unknown}'
+                    app:shortUuid="@{true}"
+                    app:value="@{data.uuid}" />
+
+                <TextView
+                    android:id="@+id/txtVaccinationDate"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:fontFamily="sans-serif"
+                    android:gravity="right"
+                    android:textColor="@color/listActivityRowLeastText"
+                    android:textSize="@dimen/listActivityRowTertiaryTextSize"
+                    app:value="@{DateFormatHelper.formatLocalDate(data.vaccinationDate)}" />
+
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/txtVaccineName"
+                    style="@style/ControlFirstOfTwoColumnsStyle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="sans-serif-medium"
+                    android:textColor="@color/listActivityRowPrimaryText"
+                    android:textSize="@dimen/listActivityRowPrimaryTextSize"
+                    app:defaultValue='@{@string/value_task_type_unknown}'
+                    app:value="@{data.VaccineName}"
+                    />
+
+                <TextView
+                    android:id="@+id/txtOtherVaccineName"
+                    style="@style/ControlSecondOfTwoColumnsStyle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="sans-serif-medium"
+                    android:gravity="right"
+                    android:textColor="@color/listActivityRowPrimaryText"
+                    android:textSize="@dimen/listActivityRowPrimaryTextSize"
+                    app:defaultValue='@{@string/value_task_type_unknown}'
+                    app:dependencyParentField="@{txtVaccineName}"
+                    app:dependencyParentValue="@{Vaccine.OTHER.toString()}"
```

Comment:
Does this work with languages other than English?

In any case, I think it makes more sense to only use one single text view here that either shows the vaccine name if it's not OTHER, or the other vaccine name otherwise.

Reply:
I updated the field. Single column style will be used.